### PR TITLE
JAVA-14176 Rename raml to raml-modules

### PR DESCRIPTION
**Pull request by anuragkumawat**

### CODE REVIEW

**Comment by asif-anwar** [comment]
retest this please

File: spring-cloud-modules/pom.xml [hunk 1]
```diff
@@ -52,7 +52,7 @@
         <module>spring-cloud-bus</module>
         <module>spring-cloud-data-flow</module>
         <module>spring-cloud-sleuth</module>
-        <module>spring-cloud-openfeign-2</module>
+        <!--<module>spring-cloud-openfeign-2</module>--> <!-- failing -->
```
Comment (dkapil) [inline spring-cloud-modules/pom.xml:55]: please revert this change, openfeign-2 should be fixed in the master branch. please update from master and PR should be running fine.
